virttest/qemu_vm.py

```diff
@@ -993,8 +993,13 @@ def add_memorys(devices, params):
             output = re.findall(r",memory-backend=mem-([\w|-]+)", machine_cmd)
             if output:
                 name = output[0]
-                backend_options = {}
+                backend_options = params.object_params("machine_mem")
```

Comment:
```suggestion
                backend_options = params.object_params(name)
```

Reply:
Since `backend_options` is an object now, we don't need to convert it again in L1006. We should use `backend_options` directly in L1007.

Reply:
thanks a lot, corrected.